### ISSUE TITLE
[BUG] fix `publish-ghcr.yml` workflow version number

### ISSUE THREAD

**Issue by Pradumnasaraf** [Pradumnasaraf/SocialSlash#17]
### Description

Update the 3 actions to the latest version

```diff
- uses: docker/login-action@f054a8b539a109f9f41c372932f1ae047eff08c9
+ docker/login-action@v2
```

```diff
- uses: docker/metadata-action@98669ae865ea3cffbcbaa878cf57c20bbf1c6c38
+ docker/metadata-action@v4
```

```diff
- uses: docker/build-push-action@ad44023a93711e3deb337508980b4b5e9bcdc5dc
+ docker/build-push-action@v4
```

FIle path -https://github.com/Pradumnasaraf/SocialSlash/blob/main/.github/workflows/publish-ghcr.yml

**Comment by tarunsamanta2k20** [comment]
I would like to work on this issue

**Comment by gauravrattan** [comment]
pls assign @Pradumnasaraf , thanks

**Comment by Pradumnasaraf** [comment]
> pls assign @Pradumnasaraf , thanks

Hey, I have already assigned you one issue on a different repo. Please let other contributors take part in it.

**Comment by gauravrattan** [comment]
@Pradumnasaraf  i had already solved the issue.

**Comment by Pradumnasaraf** [comment]
> @Pradumnasaraf i had already solved the issue.

Yes, but let other people work on the remaining issues.

**Comment by yeshwantthota** [comment]
if this issue is available, can you assign this to me?

**Comment by Pradumnasaraf** [comment]
> if this issue is available, can you assign this to me?

Go ahaed

**Comment by AyonChowdhury1307** [comment]
Hey @Pradumnasaraf can you assign me this issue?

**Comment by Pradumnasaraf** [comment]
> Hey @Pradumnasaraf can you assign me this issue?

I forget to close this issue. It's already fixed in #27. Please looks for another issue.
